### ISSUE TITLE
[BUG] Maestral hangs when syncing original Dropbox folder

### ISSUE THREAD

**Issue by remembertoremind** [SamSchott/maestral-dropbox#42]
**Describe the bug**
I installed maestral-dropbox on two machines. One had no files from dropbox, and the other already have an updated library. The one without downloaded all files with no errors. The other fails and the icon keeps looking like its updating - this error is printed:

```
[chansen@Magnus ~]$ maestral-gui
 > MaestralApiClient is ready.
2019-08-14 19:36:40 maestral.monitor INFO: Up to date
2019-08-14 19:36:40 maestral.monitor ERROR: Maestral index has become corrupted. Please rebuild.
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/maestral/monitor.py", line 397, in _load_rev_dict_from_file
    assert isinstance(rev_dict_cache, dict)
AssertionError
2019-08-14 19:36:42 maestral.monitor INFO: Indexing...
2019-08-14 19:36:44 maestral.client INFO: Indexing 500...
2019-08-14 19:36:46 maestral.client INFO: Indexing 1512...
2019-08-14 19:36:48 maestral.client INFO: Indexing 2012...
2019-08-14 19:36:49 maestral.client INFO: Indexing 2512...
2019-08-14 19:36:50 maestral.monitor INFO: Syncing...
Exception in thread MaestralFolderDownloader:
Traceback (most recent call last):
  File "/usr/lib/python3.7/threading.py", line 926, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.7/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3.7/site-packages/maestral/main.py", line 77, in folder_download_worker
    monitor.sync.get_remote_dropbox(dbx_path)
  File "/usr/lib/python3.7/site-packages/maestral/monitor.py", line 188, in wrapper
    res = func(self, *args, **kwargs)
  File "/usr/lib/python3.7/site-packages/maestral/monitor.py", line 909, in get_remote_dropbox
    success.append(self.apply_remote_changes(root_result, save_cursor=False))
  File "/usr/lib/python3.7/site-packages/maestral/monitor.py", line 974, in apply_remote_changes
    success = self._create_local_entry(folder)
  File "/usr/lib/python3.7/site-packages/maestral/monitor.py", line 188, in wrapper
    res = func(self, *args, **kwargs)
  File "/usr/lib/python3.7/site-packages/maestral/monitor.py", line 1228, in _create_local_entry
    self.set_local_rev(entry.path_display, "folder")
  File "/usr/lib/python3.7/site-packages/maestral/monitor.py", line 478, in set_local_rev
    if rev == self._rev_dict_cache.get(dbx_path, None):
AttributeError: 'int' object has no attribute 'get'
```

**To Reproduce**
- Install as specified in the readme. Start maestral-gui.

**Expected behaviour**
Mastral-gui uploads latest files goes into sleep.

**System (please complete the following information):**
 - OS: Newest arch linux 5.2.8-arch1-1-ARCH
 - i3-gaps
 - Qt: 5.13.0

**Additional context**

**Comment by SamSchott** [comment]
Are you trying to sync a Dropbox folder that has already been created by the official client? This is unfortunately not supported by Maestral at the moment since Dropbox uses a different approach to keep track of file versions. For now, you should be able fix the issue by deleting the "hidden" file ".dropbox" in your Dropbox folder and restarting Maestral.

The next version of Maestral will default to using the folder "Dropbox (Maestral)" instead to prevent conflicts.

**Comment by remembertoremind** [comment]
> Are you trying to sync a Dropbox folder that has already been created by the official client?
Yup, that was my intention.

> For now, you should be able fix the issue by deleting the hidden file ".dropbox" in your Dropbox folder and restarting Maestral.
I had a `.dropbox-cache` and deleting it did nothing. Deleting the whole folder made mastral-gui work as expedted. 

Thanks for the fast replies :)

**Comment by SamSchott** [comment]
This should b fixed in v0.3.0.
